Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonForm.cs

```diff
@@ -1191,6 +1210,21 @@ public SystemMenuValues SystemMenuValues
 
     /// <summary>
     /// Gets access to the system menu for advanced customization.
+    /// 
+    /// IMPORTANT: This property returns NULL in design mode to prevent designer interference.
```

Comment:
Some of these can go into <Remoarks> and <notes> sections, so that when intellisense first pops up it doesn;t flood the screen

Reply:
Fixed